packages/firestore/src/lite-api/database.ts

```diff
@@ -325,6 +325,11 @@ export function connectFirestoreEmulator(
   } = {}
 ): void {
   firestore = cast(firestore, Firestore);
+  let ssl = false;
+  if (/^https:\/\//.test(host)) {
+    ssl = true;
+    host = host.substring(8);
+  }
```

Comment:
I think I would stick with the existing FirestoreSettings API and keep `host` the host name only. Add another option `ssl` to `connecteFirestoreEmulator(...)`. Default that to false. 

I've seen some customer confusion around including scheme on the `host` value. If we can keep things consistent, that will reduce further confusion.